Microsoft.Azure.CosmosRepository/src/Providers/ICosmosContainerSyncContainerPropertiesProvider.cs

```diff
@@ -15,5 +17,7 @@ public interface ICosmosContainerSyncContainerPropertiesProvider
         /// <remarks>If the SyncAllContainerProperties is set to true this will override any specific containers configuration.</remarks>
         /// <returns>Whether or not to sync container properties</returns>
         bool GetWhetherToSyncContainerProperties<TItem>() where TItem : IItem;
```

Comment:
Might be a good use case for default interface members.
```suggestion
        bool GetWhetherToSyncContainerProperties<TItem>() where TItem : IItem =>
            GetWhetherToSyncContainerProperties(typeof(TItem));
```

Reply:
I like this.

Reply:
@IEvangelist this can be done but only if we bump the package up to `netstandard2.1` which would mean we drop .NET framework support, what are your thoughts on this?

Reply:
Yeah, it's not worth it. Using a somewhat cool feature vs. dropping support for an entire framework. Let's not do that 😄

Reply:
Cool, I thought the same